### PR TITLE
feat: New workflow for big xmluploads: add commands `upload-files` and `ingest-files` (DEV-3905)

### PR DESCRIPTION
**Pull request by jnussbaum**

### CODE REVIEW

File: src/dsp_tools/commands/ingest_xmlupload/upload_files/upload_files.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    paths = _get_validated_paths(root)
+    print(f"Found {len(paths)} files to upload onto server {creds.dsp_ingest_url}.")
+    logger.info(f"Found {len(paths)} files to upload onto server {creds.dsp_ingest_url}.")
+
+    con: Connection = ConnectionLive(creds.server)
+    con.login(creds.user, creds.password)
+    ingest_client = BulkIngestClient(creds.dsp_ingest_url, con.get_token(), shortcode, imgdir)
+
+    failures: list[UploadFailure] = []
+    progress_bar = tqdm(sorted(paths), desc="Uploading files", unit="files")  # sorting is for testability
```
Comment (BalduinLandolt) [inline src/dsp_tools/commands/ingest_xmlupload/upload_files/upload_files.py:45]: I don't think this comment is accurate anymore, right? can the `sorted()` be removed?

**Review by BalduinLandolt — APPROVED** [review]

**Comment by seakayone** [comment]
Please replace testdata/invalid-testdata/bitstreams/test.gif with an image which is free of copyright.

I found this image on flickr with all rights reserved: https://www.flickr.com/photos/186119147@N03/49249898003/

**Comment by jnussbaum** [comment]
> Please replace testdata/invalid-testdata/bitstreams/test.gif with an image which is free of copyright.
> 
> I found this image on flickr with all rights reserved: https://www.flickr.com/photos/186119147@N03/49249898003/

Thank you very much for pointing this out. I will do that in https://linear.app/dasch/issue/DEV-3906/replace-testdata-which-is-not-copyright-free

File: src/dsp_tools/commands/ingest_xmlupload/upload_files/upload_files.py (hunk 1 above)
Comment (jnussbaum) [reply src/dsp_tools/commands/ingest_xmlupload/upload_files/upload_files.py]: correct, I have removed it.

**Pull request merged**